src/modules/75-cf/components/NoEnvironment/NoEnvironment.tsx

```diff
@@ -6,46 +6,26 @@
  */
 
 import React from 'react'
-import { Container, Heading, Text } from '@wings-software/uicore'
-import type { ContainerProps } from '@wings-software/uicore/dist/components/Container/Container'
-import { useStrings } from 'framework/strings'
+import type { ContainerProps } from '@harness/uicore/dist/components/Container/Container'
+import { String, useStrings } from 'framework/strings'
 import EmptyContent from './EmptyContent.svg'
 import EnvironmentDialog, { EnvironmentDialogProps } from '../CreateEnvironmentDialog/EnvironmentDialog'
+import { NoData } from '../NoData/NoData'
 
 export interface NoEnvironmentProps extends ContainerProps {
   onCreated: EnvironmentDialogProps['onCreate']
 }
```

Comment:
Looks like you're not using any of the members from `ContainerProps`, so you can simplify.
```suggestion
export interface NoEnvironmentProps {
  onCreated: EnvironmentDialogProps['onCreate']
}
```